Auth0/OAuth2Grant.swift

```diff
@@ -67,50 +81,99 @@ struct PKCE: OAuth2Grant {
     let defaults: [String: String]
     let verifier: String
     let responseType: [ResponseType]
-
-    init(authentication: Authentication, redirectURL: URL, generator: A0SHA256ChallengeGenerator = A0SHA256ChallengeGenerator(), reponseType: [ResponseType] = [.code], nonce: String? = nil) {
-        self.init(authentication: authentication, redirectURL: redirectURL, verifier: generator.verifier, challenge: generator.challenge, method: generator.method, responseType: reponseType, nonce: nonce)
+    let leeway: Int
+    let maxAge: Int?
+
+    init(authentication: Authentication,
+         redirectURL: URL,
+         generator: A0SHA256ChallengeGenerator = A0SHA256ChallengeGenerator(),
+         reponseType: [ResponseType] = [.code],
+         nonce: String? = nil,
+         leeway: Int,
+         maxAge: Int? = nil) {
+        self.init(authentication: authentication,
```

Comment:
**Nit:** Indent?

Reply:
It's 4 spaces to the right of `init`, so it's correct.